core/templates/services/context.service.spec.ts

```diff
@@ -160,6 +181,17 @@ describe('Context service', () => {
       () => {
         expect(ecs.canAddOrEditComponents()).toBe(true);
       });
+
+    it('should correctly affirm that page contains reference to skills',
+      () => {
+        expect(ecs.getPageContext()).toBe('topic_editor');
+        expect(ecs.canEntityReferToSkills()).toBe(true);
+      });
+
+    it('should affirm the page context other than exploration editor ' +
+      'and player', ()=> {
+      expect(ecs.isInExplorationContext()).toBe(false);
```

Comment:
I don't see where the context is being set. Please set a non exploration context in this test and then check that `isInExplorationContext()` returns false.

Reply:
Done!

Reply:
Done!